docs/source/getting-started/quickstart.rst

```diff
@@ -268,7 +263,7 @@ Develop with DAML Studio
 
 Take a look at the DAML that specifies the contract model in the quickstart application. The core template is ``Iou``.
 
-#. Open :doc:`DAML Studio </daml/daml-studio>`, a DAML IDE based on VS Code, by running ``da studio`` from the root of your project.
+#. Open :doc:`DAML Studio </daml/daml-studio>`, a DAML IDE based on VS Code, by running ``daml studio`` from the root of your project.
```

Comment:
I don't know whether/how to improve this, but it's not accurate to say that `daml studio` needs to be run from the root of the project, it only needs to be run inside the project (not necessarily the root), and that will cause VSCode to be opened at the root of the project.

```suggestion
#. Open :doc:`DAML Studio </daml/daml-studio>`, a DAML IDE based on VS Code, by running ``daml studio`` from the root of your project.
```